### PR TITLE
moved --tag to be common option not build group related one.

### PR DESCRIPTION
**Pull request by dbecvarik**

### CODE REVIEW

File: concreate/cli.py [hunk 1]
```diff
@@ -64,12 +64,8 @@ def parse(self):
...
                                  dest='build_tags',
                                  action='append',
-                                 help='tag to assign to the built image, can be used multiple times')
-
-        build_group.add_argument('--tag',
-                                 dest='tags',
-                                 action='append',
-                                 help='tag used to build/test the image, can be used multiple times')
+                                 help='tag to assign to the built image, '
+                                 'can be used multiple times')
```
Comment (goldmann) [inline concreate/cli.py]: This line breaking is too eager.

Comment (dbecvarik) [reply concreate/cli.py]: its not, 101 characters is more than 100 allowed for us

Comment (goldmann) [reply concreate/cli.py]: This is I don't like it. At all.

**Comment by goldmann** [comment]
There is a conflict.

File: concreate/cli.py (hunk 1 above)
Comment (dbecvarik) [reply concreate/cli.py]: Can you suggest better breaking?

Comment (goldmann) [reply concreate/cli.py]: Sure, not breaking it :)

Comment (dbecvarik) [reply concreate/cli.py]: I can do something like:
``` python
        build_group.add_argument('--build-tag',
                                 dest='build_tags',
                                 action='append',
                                 help=('tag to assign to the built image, '
                                       'can be used multiple times'))
```

which is maybe indented better?

**Pull request merged**